sdk/storage/Azure.Storage.Blobs/tests/TestExtensions.cs

```diff
@@ -0,0 +1,55 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Azure.Core;
+using Azure.Storage.Blobs;
+using Azure.Storage.Blobs.Models;
+using Azure.Storage.Blobs.Specialized;
+
+namespace Azure.Storage
```

Comment:
Shouldn't this be in Test namespace?

Reply:
It doesn't matter much and it's an existing partial class.  I'm trying to touch as few files as possible with all the big changes in flight.